DigitalOffice/Services/CompanyService/Database/CompanyServiceDbContext.cs

```diff
@@ -17,6 +17,7 @@ public CompanyServiceDbContext (DbContextOptions<CompanyServiceDbContext> option
         public DbSet<DbPosition> Positions { get; set; }
         public DbSet<DbCompany> Companies { get; set; }
         public DbSet<DbDepartment> Departments { get; set; }
+        public DbSet<DbCompanyUser> CompanyUser { get; set; }
```

Comment:
CompaniesUsers?

Reply:
If I write like this, an exception will be thrown with the following message "invalid name for CompaniesUsers".

Reply:
renamed